src/DOMTestCase.php

```diff
@@ -22,137 +20,5 @@
-    {
-        self::assertSelectEquals(
-            $selector, "regexp:$pattern", $count, $actual, $message, $isHtml
-        );
-    }
-
-    /**
-     * assertSelectEquals("#binder .name", "Chuck", true,  $xml);  // any?
-     * assertSelectEquals("#binder .name", "Chuck", false, $xml);  // none?
-     *
-     * @param string                $selector
-     * @param string|null           $content
-     * @param integer|boolean|array{"<"?: int, ">"?: int, "<="?: int, ">="?: int} $count
-     * @param mixed                 $actual
-     * @param string                $message
-     * @param boolean               $isHtml
-     * @since Method available since Release 1.0.0
-     *
-     * @throws \PHPUnit\Framework\Exception
-     *
-     * @return void
-     */
-    public static function assertSelectEquals($selector, $content, $count, $actual, $message = '', $isHtml = true)
-    {
-        $crawler = new Crawler;
-
-        if ($actual instanceof \DOMDocument) {
-            $crawler->addDocument($actual);
-        } else if ($isHtml) {
-            $crawler->addHtmlContent($actual);
-        } else {
-            $crawler->addXmlContent($actual);
-        }
-
-        $crawler = $crawler->filter($selector);
-
-        if (is_string($content)) {
-            $crawler = $crawler->reduce(function (Crawler $node, $i) use ($content) {
-                $text = $node->text(null, true);
-
-                if ($content === '') {
-                    return $text === '';
-                }
-
-                if (preg_match('/^regexp\s*:\s*(.*)/i', $content, $matches)) {
-                    return (bool)preg_match($matches[1], $text);
-                }
-
-                return strstr($text, $content) !== false;
-            });
-        }
-
-        $found = count($crawler);
-
-        if (is_numeric($count)) {
-            self::assertEquals($count, $found, $message);
-        } else if (is_bool($count)) {
-            $found = $found > 0;
-
-            if ($count) {
-                self::assertTrue($found, $message);
-            } else {
-                self::assertFalse($found, $message);
-            }
-        } else if (is_array($count) &&
-            (isset($count['>']) || isset($count['<']) ||
-                isset($count['>=']) || isset($count['<=']))) {
-
-            if (isset($count['>'])) {
-                self::assertTrue($found > $count['>'], $message);
-            }
-
-            if (isset($count['>='])) {
-                self::assertTrue($found >= $count['>='], $message);
-            }
-
-            if (isset($count['<'])) {
-                self::assertTrue($found < $count['<'], $message);
-            }
-
-            if (isset($count['<='])) {
-                self::assertTrue($found <= $count['<='], $message);
-            }
-        } else {
-            throw new \PHPUnit\Framework\Exception('Invalid count format');
-        }
-    }
+    use DOMTestTrait;
```

Comment:
Moving the source-of-truth to the trait itself reduces code duplication while keeping BC